requirements.txt

```
@@ -15,7 +15,7 @@ pathlib==1.0.1; python_version < "3.4"
 jsonschema>=2.6.0,<3.1.0
 # Development dependencies
 cython>=0.25
-pytest>=4.0.0,<4.1.0
+pytest>=5.2.0,<5.3.0
```

Apparently this is the last version that supports Python 2?

```suggestion
pytest>=4.6.5```